### DISCUSSION TITLE
Alfred执行系统lock命令不起作用

### DISCUSSION THREAD

**Discussion by alanhe421** [alanhg/alfred-workflows#119] [Q&A 常见问题]
![image](https://user-images.githubusercontent.com/9245110/43367256-cf3dfc88-937c-11e8-83b8-9b956c76c093.png)
我在使用lock时发现会不起作用，原来是QQ中的快捷键与之冲突了。
索性直接去掉这几个快捷键设定，发现OK了
![image](https://user-images.githubusercontent.com/9245110/43367267-f7b1f9c6-937c-11e8-88ba-30f14e9f6c66.png)

**Comment by KingCoCos** [comment]
👍🏻

**Comment by winterwd** [comment]
我也碰到了，但是我没有安装QQ😢

**Comment by alanhe421** [comment]
> 我也碰到了，但是我没有安装QQ😢

看下搜狗输入法之类的，将一些不用的热键都取消下绑定

**Comment by finviman** [comment]
alfred是通过快捷键触发的吗，但我按ctrl+cmd+q是可以锁屏的，但是alfred命令不行。

**Reply by yuanquanke66** [reply]
我的也是

**Reply by alanhe421** [reply]
> alfred是通过快捷键触发的吗，但我按ctrl+cmd+q是可以锁屏的，但是alfred命令不行。

嗯，目前了解Alfred的实现原理是热键。https://www.alfredforum.com/topic/15395-lock-doesnt-work-when-qqan-instant-communication-app-in-china-is-running/?tab=comments#comment-78821

**Reply by shuifa** [reply]
> > alfred是通过快捷键触发的吗，但我按ctrl+cmd+q是可以锁屏的，但是alfred命令不行。
> 
> 嗯，目前了解Alfred的实现原理是热键。https://www.alfredforum.com/topic/15395-lock-doesnt-work-when-qqan-instant-communication-app-in-china-is-running/?tab=comments#comment-78821

那这个现在有什么解决办法吗？使用快捷键可以锁屏，alfred的lock无效

**Reply by alanhe421** [reply]
个人觉得2个办法
1. 找到冲突的软件热键设定
2. 放弃Alfred本身的lock command，自己写个keyword workflow。
比如，具体执行command不行就换成run script ` pmset displaysleepnow`之类的，本质就是自己搞了个lock workflow。
![image](https://github.com/alanhg/alfred-workflows/assets/9245110/b52407c5-6385-4ec0-96e2-fad14142760d)

**Comment by pprain** [comment]
请问怎么发现是和QQ冲突的呢？
